dagshub/fastai/logger.py

```diff
@@ -17,6 +21,9 @@ class DAGsHubLogger(Callback):
     learn.fit(..., cbs=DAGsHubLogger())
     ```
     """
+    
+    "Saves model topology, losses & metrics"
+    remove_on_fetch, order = True, Recorder.order+1
```

Comment:
This is FastAI black magic. Not sure what it is and why it's necessary, but you must have this otherwise logging doesn't work